catalog/ui/catalog-ui-search/src/main/java/org/codice/ddf/catalog/ui/forms/filter/VisitableJsonElementImpl.java

```diff
@@ -147,68 +178,52 @@ private VisitableJsonElementImpl(final FilterNode node) {
           prop,
           val,
           distance);
-      this.value = wrap(prop, val, distance);
+      this.value =
+          Arrays.asList(
+              new VisitableJsonElementImpl(FilterVisitor2::visitString, prop),
+              new VisitableJsonElementImpl(FilterVisitor2::visitDoubleType, distance),
+              new VisitableJsonElementImpl(
+                  FilterVisitor2::visitLiteralType, Collections.singletonList(val)));
       return;
     }
 
     if (val == null) {
       LOGGER.trace("Found JSON nil node [{}] for property [{}]", name, prop);
-      this.value = new VisitableJsonElementImpl(prop);
+      this.value = new VisitableJsonElementImpl(FilterVisitor2::visitString, prop);
       return;
     }
 
-    this.value = wrap(prop, val);
-  }
-
-  private VisitableJsonElementImpl(final String operator, final Object value) {
-    this.name = operator;
-    this.value = value;
-    this.visitMethod = VISIT_METHODS.get(operator);
-    if (this.visitMethod == null) {
-      throw new FilterProcessingException(
-          "Encountered an unexpected or unsupported type: " + operator);
+    if ("BETWEEN".equals(node.getOperator()) || "LIKE".equals(node.getOperator())) {
+      LOGGER.trace(
+          "Found JSON map-visitable node [{}] for property [{}] and value [{}]", name, prop, val);
```

Comment:
Yeah for the `LIKE` case that would make sense. Not so much for the `BETWEEN` because the UI uses that as a "Range" operator for numerics, and you can't input plaintext into those fields, or copy-paste values.

Reply:
But a user could call the endpoint directly with another value if we are not validating on the backend endpoint. We should always be validating on the backend for that reason but I suspect we probably are not here.

Reply:
Since nothing in forms uses `LogSanitizer` and almost all the logs are `trace` logs which are off in production, we decided to call it out of scope for this PR. Also, nothing automatically processes the form logs, so the risk is pretty minimal.